### PR TITLE
Make Debug-mode notification link to debug settings page

### PR DESCRIPTION
**Pull request by panploenes**
| Questions | Answers |
| --- | --- |
| Branch? | Develop |
| Description? | Added a href so that the Debug notification links to the correlating settings page. |
| Type? | improvement |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | - |
| How to test? | Access the backoffice and activate the debug mode. Then click on the Debug notification in the top right corner. |

### CODE REVIEW

File: admin-dev/themes/default/template/header.tpl [hunk 1]
```diff
@@ -301,8 +301,9 @@
               {if isset($debug_mode) && $debug_mode == true}
                 <span class="shop-state" id="debug-mode">
                   <i class="material-icons">bug_report</i>
-                  <span class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
-                    title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in debug mode.'}</strong></p><p class='text-left'>{l s='All the PHP errors and messages are displayed. When you no longer need it, [1]turn off[/1] this mode.' tags=['<strong>']}</p>">{l s='Debug mode'}</span>
+                  <a class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/template/header.tpl]: this is not valid HTML

File: admin-dev/themes/default/template/header.tpl [hunk 2]
```diff
@@ -301,8 +301,9 @@
...
+                    title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in debug mode.'}</strong></p><p    class='text-left'>{l s='All the PHP errors and messages are displayed. When you no longer need it, [1]turn off[/1] this mode.' tags=['<strong>']}</p>" href="{$link->getAdminLink('AdminPerformance')|escape:'html':'UTF-8'}">{l s='Debug mode'}
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/template/header.tpl]: to be honest this was not valid HTML too :(

File: admin-dev/themes/default/template/header.tpl (hunk 1 above)
Comment (panploenes) [inline admin-dev/themes/default/template/header.tpl]: Just copy pasted and adapted from the Maintance Mode section below.

Comment (mickaelandrieu) [inline admin-dev/themes/default/template/header.tpl]: still, its not valid HTML :)

Comment (kpodemski) [inline admin-dev/themes/default/template/header.tpl]: uhh, i think that it should be accepted as is and changed globally after that...

**Comment by panploenes** [comment]
I gonna close this PR for now. Unless someone can propose the proper HTML.

**Comment by antoin-m** [comment]
Hey @panploenes, thank you for your contribution!

Unfortunately we will have to rework this part in order to produce valid HTML. I'll close this PR because it probably won't be working using the new system.
Michel.

**Pull request closed without merging**